src/normalize/entities.ts

```diff
@@ -187,21 +187,24 @@ export const SendMessageInfo = new Entity({
 	m: ['message', SendMessageFields]
 });
 
-export const MessageInfo = new Entity({
-	...commonMailItemFields
+export const MessageHitInfo = new Entity({
+	...commonMailItemFields,
+	cm: 'messageMatchedQuery'
 });
 
+export const MessageInfo = new Entity(commonMailItemFields);
+
 export const Conversation = new Entity({
 	...commonMailItemFields,
 	n: 'numMessages',
 	m: ['messages', MessageInfo],
 	u: 'unread'
 });
 
-export const SearchConversation = new Entity({
+export const ConversationHitInfo = new Entity({
 	...commonMailItemFields,
 	n: 'numMessages',
-	m: ['messagesMetaData', MessageInfo],
```

Comment:
I've renamed this property back to messages, I'm not sure I totally understood the change from https://github.com/Zimbra/zm-api-js-client/pull/117 but it looked like the conflicting types were overwriting each other (`SearchResponse` and `GetConvResponse`). This shouldn't happen anymore since there are now three distinct conversation types - `ConversationInfo`, `ConversationHitInfo`, and `NestedSearchConversation` - so they should not overwrite one another.